### ISSUE TITLE
Error when checking model input: the list of Numpy arrays that you are passing to your model is not the size the model expected. Expected to see 1 array(s), but instead got the following list of 12331 arrays

### ISSUE THREAD

**Issue by nihayahzahra** [keras-team/keras#15663]
I want build model in keras with LSTM model. The code is given below

`#build the model: 2 stacked LSTM
print('Build model...')
model = Sequential()
model.add(LSTM(512, return_sequences=True, input_shape=(maxlen,len_vocab)))
model.add(Dropout(0.2))
model.add(LSTM(512, return_sequences=False))
model.add(Dropout(0.2))
model.add(Dense(2))
model.add(Activation('softmax'))
model.compile(loss='categorical_crossentropy', optimizer='adam',metrics=['accuracy'])

test_X = []
test_Y = []
trunc_test_name = [str(i)[0:maxlen] for i in test.name]
for i in trunc_test_name:
    tmp = [set_flag(char_index[j]) for j in str(i)]
    for k in range(0,maxlen - len(str(i))):
        tmp.append(set_flag(char_index["END"]))
    test_X.append(tmp)
for i in test.m_or_f:
    if i == 'm':
        test_Y.append([1,0])
    else:
        test_Y.append([0,1])

print(np.asarray(test_X).shape)
print(np.asarray(test_Y).shape)

batch_size=1000
model.fit(train_X, train_Y,batch_size=batch_size,validation_data=(test_X, test_Y))`

The above program is giving the following error. 
Error when checking model input: the list of Numpy arrays that you are passing to your model is not the size the model expected. Expected to see 1 array(s), but instead got the following list of 12331 arrays : [array([[0., 0., 0., ..., 0., 0., 0.],
       [0., 0., 0., ..., 0., 0., 0.],
       [0., 0., 0., ..., 0., 0., 0.],
       ...,
       [0., 0., 0., ..., 0., 0., 0.],
       [0., 0., 0., ..., 0., 0., 0....

**Comment by chunduriv** [comment]
@nihayahzahra, Your input arrays must be NumPy arrays, not lists. 

Could you please refer to the [issue](https://stackoverflow.com/questions/42596057/keras-error-expected-to-see-1-array), [issue2](https://github.com/keras-team/keras/issues/9475#issuecomment-472662095) and let us know if it helps ? Thanks!
